From 1d27fc888caabeec2e8bcc5eaf63b9cf7b1cde89 Mon Sep 17 00:00:00 2001
From: Badhrinath S <badhrina@qti.qualcomm.com>
Date: Sat, 31 Jan 2026 14:39:30 +0530
Subject: [PATCH] arm: dts: ipq9650: Add RDP build support

This change adds the build support for the following RDP's:
RDP488, RDP489, RDP506

Change-Id: Id40157ee7d95a142f25569908b9e9ef6809971f4
Signed-off-by: Badhrinath S <badhrina@qti.qualcomm.com>
---
 arch/arm/dts/Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 0ebe954fa16..c03fd2e04cf 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -12,6 +12,11 @@ dtb-$(CONFIG_TARGET_IPQ5210) += ipq5210-emulation.dtb \
 				ipq5210-db-hm01.1.dtb \
 				ipq5210-db-hm02.1.dtb
 
+dtb-$(CONFIG_TARGET_IPQ9650) += ipq9650-emulation.dtb \
+                                ipq9650-rdp488.dtb \
+                                ipq9650-rdp489.dtb \
+                                ipq9650-rdp506.dtb
+
 dtb-$(CONFIG_TARGET_SMARTWEB) += at91sam9260-smartweb.dtb
 dtb-$(CONFIG_TARGET_TAURUS) += at91sam9g20-taurus.dtb
 dtb-$(CONFIG_TARGET_CORVUS) += at91sam9g45-corvus.dtb
-- 
2.34.1

